### PR TITLE
CAVAU

### PR DESCRIPTION
**Pull request by matthaus000**

### CODE REVIEW

File: registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "owner": "ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a",
+  "name": "Zimzalabim",
+  "ticker": "ZIMZA",
+  "homepage": "https://cardanofoundation.org",
```
Comment (mmahut) [inline registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json]: I suggest you to change this.

Comment (matthaus000) [reply registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json]: hey mmahut I am unsure what part is the issue in the above code.

Comment (matthaus000) [reply registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json]: > I suggest you to change this.

hey mmahut I am unsure what part is the issue in the above code.

Comment (mmahut) [reply registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json]: This looks like your pool would be run by CF.

**Comment by matthaus000** [comment]
check failing. Need review?

**Comment by matthaus000** [comment]
changed site url

**Pull request merged**